.github/README.md

```diff
@@ -0,0 +1,50 @@
+# Hedera Smart Contracts Workflows
+
+A detailed overview of how you can use hedera-smart-contract's github workflows for testing purposes. Besides examples, there will be explanations of what is happening under the hood.
+
+## Actions
+
+### Migration Testing
+
+When there is a new version released, no matter if it is network-node, mirror-node, or relay, we want to make sure the entire ecosystem is working as expected and there are not any critical regressions introduced. Another concern with switching the network node's versions is "the migration" that is running under the hood. It occurs every time a node has been started and detects if there is data for migration. The migration testing workflow offers this type of testing where we can choose versions for pre/post image tags.
```

Comment:
I feel like we can drop the `Another concern with switching...` to a new paragraph for easier reading.